appbars/README.md

```diff
@@ -0,0 +1,173 @@
+# AppBar components
+
+## AppBar design specs
+
+You can find the design specs on [www.decathlon.design](https://www.decathlon.design)
+
+## Usage with Vitamin Theme
+
+By inheriting the Vitamin theme in your app, you just have to use the default material
+component or the Vitamin subclass component.
+
+```kotlin
+implementation("com.decathlon.vitamin:appbars:<version>")
+```
+
+### TopBar
+
+```xml
+<com.decathlon.vitamin.appbars.VitaminTopBar
+    style="?attr/toolbarStyle"
```

Comment:
Is that mandatory? The VitaminTopBar does not already set that?

Reply:
Yes... :(